types/mithril/tslint.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
All right, extending "../tslint.json" now and cleaned up some code style issues.